INSTALL.md

```diff
@@ -11,16 +11,21 @@ following sections describe them for the supported platforms.
 * GNU Automake
 * GNU Libtool
 * C compiler
-* C Library Development Libraries and Header Files
+
+### C Library Development Libraries and Header Files
 * pkg-config
 * uriparser
 * libgcrypt
 
 The following are dependencies only required when building test suites.
 * Integration test suite (see ./configure option --with-simulatorbin):
-** OpenSSL
+
+   OpenSSL libraries and header files
+
 * Unit test suite (see ./configure option --enable-unit):
-** cmocka unit test framework
+
+   cmocka unit test framework
```

Comment:
same as last one

Reply:
Same as above. The “**” markup for nested indentation is not supported—at least for the GitHub flavor of markup language.   So you get this confusing display:

Unit test suite (see ./configure option --enable-unit): ** cmocka unit test framework Most users will not need to install these dependencies.

Take a look yourself at
https://github.com/tpm2-software/tpm2-tss/blob/master/INSTALL.md